### ISSUE TITLE
Amazon Q chat input does not work well with IME enabled

### ISSUE THREAD

**Issue by tmokmss** [aws/aws-toolkit-vscode#4125]
## Problem
When I type something into Amazon Q Chat with [IME](https://learn.microsoft.com/en-us/windows/apps/design/input/input-method-editors) enabled, it will send my message as soon as I hit enter, where I expect not to send message but to continue to input following characters. 

I think we need to specially handle `keydown` events using `event.isComposing` property ([ref](https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event#ignoring_keydown_during_ime_composition)).

```js
eventTarget.addEventListener("keydown", (event) => {
  if (event.isComposing || event.keyCode === 229) {
    return;
  }
  // do something
});
```

I know Amazon Q currently supports only English, but someday it will become problem when it starts to support CJK languages.

## Steps to reproduce the issue
Enable Japanese keyboard, and type `a` -> `i`-> `Enter`. 

Current behavior: `あい` is sent to chat.

<img width="489" alt="image" src="https://github.com/aws/aws-toolkit-vscode/assets/7490655/9ddbe5ac-1771-4774-9fba-e7c0800a9d96">

## Expected behavior
Expected behavior: `あい` is confirmed but not sent to chat. Instead we should be able to continue to write message.

## System details (run the `AWS: About Toolkit` command)

-   OS: macOS
-   Visual Studio Code version: 1.84.2
-   AWS Toolkit version: 2.1.0

**Comment by eccyan** [comment]
I face the same problem, please fix for non-english users.

**Comment by dogusata** [comment]
Thanks for letting us know. We'll enable IME check on the input for Q Chat.

**Comment by justinmk3** [comment]
Fixed in AWS Toolkit 2.15.0
